### ISSUE TITLE
How to use use the fscheck "Observing Test Case Distribution" features with expecto

### ISSUE THREAD

**Issue by Vidarls** [haf/expecto#221]
I have been trying to use the fscheck features for observing test case distribution with expecto, and I can't seem to get any distribution output. 

Is this a feature that is not supported, or am I doing-it-wrong(tm)

ref: https://fscheck.github.io/FsCheck/Properties.html#Observing-Test-Case-Distribution

This works as expected:

```
let ``a test property`` (testString:string) =
    not (String.IsNullOrEmpty testString) ==> lazy (true |> Prop.collect (testString.Length))
Check.Quick ``a test property`` 
```

This does not give any distribution output:

```
let myTest = 
    testPropertyWithConfig config "A test property" <|
            fun (testString:string) ->
                not (String.IsNullOrEmpty testString) ==> lazy (true |> Prop.collect (testString.Length))

runTests myTest
```

**Comment by AnthonyLloyd** [comment]
I don't think its supported at the moment. I'm trying to work out if its just ignore a string function in the config or runner or if its something more fundamental.

Will need to look at this when I have a little more time.

**Comment by Vidarls** [comment]
Is it possible the same mechanism might be used for labels too.  (https://fscheck.github.io/FsCheck/Properties.html#And-Or-and-Labels)
I have been unable to get label output when using expecto.

**Comment by AnthonyLloyd** [comment]
Yes. I'll look at this a bit more tonight.

**Comment by AnthonyLloyd** [comment]
I know how this is working now. I've put in stamps and label for failing tests. I will need to have a think about if it makes sense to do this for passing tests.

https://github.com/haf/expecto/commit/eef3f2ea91c193a2612d18c6c4d494933f1e9337

**Comment by AnthonyLloyd** [comment]
Just released this in Expecto v7.0.0. Let me know how it looks for you. I'll keep this open a while.
